lib/main_common.pm

```diff
@@ -2400,117 +2400,109 @@ sub load_virtualization_tests {
-        loadtest 'virtualization/universal/hotplugging_vCPUs';    # Add and remove guests vCPU
-        loadtest 'virtualization/universal/hotplugging_memory';    # Live memory change of guests
-        loadtest 'virtualization/universal/hotplugging_cleanup';    # Restore guests properties
-    }
-
-    if (check_var('VIRT_PART', 'networking')) {
-        loadtest "virt_autotest/libvirt_host_bridge_virtual_network";
-        loadtest "virt_autotest/libvirt_nated_virtual_network";
-        loadtest "virt_autotest/libvirt_isolated_virtual_network";
-    }
-
-    if (check_var('VIRT_PART', 'irqbalance')) {
-        loadtest "virt_autotest/xen_guest_irqbalance";
-    }
-
-    if (check_var('VIRT_PART', 'snapshots')) {
-        loadtest "virt_autotest/virsh_internal_snapshot";
-        loadtest "virt_autotest/virsh_external_snapshot";
-
+    if (check_var('ENABLE_HOST_INSTALLATION', 1)) {
+        if (get_var('AUTOYAST')) {
+            loadtest "autoyast/prepare_profile";
+        }
+        loadtest "boot/boot_from_pxe";
+        if (get_var('AUTOYAST')) {
+            loadtest "autoyast/installation";
+        } else {
+            loadtest "installation/welcome";
+            loadtest "installation/scc_registration";
+            loadtest "installation/addon_products_sle";
+            loadtest "installation/system_role";
+            loadtest "installation/partitioning";
+            loadtest "installation/partitioning_firstdisk";
+            loadtest "installation/partitioning_finish";
+            loadtest "installation/installer_timezone";
+            loadtest "installation/user_settings";
+            loadtest "installation/user_settings_root";
+            loadtest "installation/resolve_dependency_issues";
+            loadtest "installation/installation_overview";
+            loadtest "installation/disable_grub_timeout";
+            loadtest "installation/start_install";
+            loadtest "installation/await_install";
+            loadtest "installation/reboot_after_installation";
+        }
     }
 
-    if (check_var('VIRT_PART', 'storage')) {
-        loadtest 'virtualization/universal/storage';    # Storage pool / volume test
-    }
+    if (check_var('ENABLE_VM_INSTALL', 1)) {
+        loadtest "virt_autotest/login_console";
+        loadtest "virtualization/universal/prepare_guests";
+        loadtest "virtualization/universal/waitfor_guests";
+        if (check_var('PATCH_WITH_ZYPPER', 1)) {
+            loadtest "virtualization/universal/patch_and_reboot";
+            if (my $update_package = get_var('UPDATE_PACKAGE')) {
+                if ($update_package eq 'kernel-default') {
+                    loadtest "virt_autotest/login_console";
+                    loadtest "virtualization/universal/list_guests";
+                    loadtest "virtualization/universal/patch_guests";
+                } elsif ($update_package eq 'xen' || $update_package eq 'qemu') {
+                    loadtest "virt_autotest/login_console";
```

Comment:
I just noticed after looking into details -- this `loadtest "virt_autotest/login_console";` shows several times, but actually it is  only needed *once* after host installation (proved in OSD development product test), no matter how many features or what features are tested, and no matter debug test or official test. 
I think putting it just after line 2447 will be good enough.

Reply:
Before login_console, there is a reboot: loadtest "virtualization/universal/patch_and_reboot". So I think still need new login_console, what do you think of it?

Reply:
Likely. I tried to find a job with patch_and_reboot to see if it is needed, but I do not find one. Suggest to have a try.

Reply:
Temporarily keep this part as it is, consistent with the original. If improvements are needed, I can make adjustments later. This approach is relatively stable now.